### PR TITLE
adjust authors to be more neutral about ex comtravos

### PR DESCRIPTION
**Pull request by sebastianmika**

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ published: true
 		<div class="inner-wrap">
 			<div id="content" class="page-content" itemprop="articleBody">
                 <footer class="page-footer">
-                <h2>The Comtravo tech blog is written by</h2>
+                <h4>The Comtravo tech blog is written by many great minds. Not surprinsgly many of them are current or former Comtravo employees.</h4>
```
Comment (bhuvinth) [inline about.md:28]: Just fixing a typo: 
```suggestion
                <h4>The Comtravo tech blog is written by many great minds. Not surprisingly many of them are current or former Comtravo employees.</h4>
```

**Comment by utvara** [comment]
👍💪

**Review by davidsbatista — APPROVED** [review]
Looks good :)

File: _data/authors.yml [hunk 2]
```diff
@@ -44,20 +44,20 @@ marko_schilde:
     name: Marko Schilde
     email: tech@comtravo.com
     web: https://www.github.com/mschilde
-    role: Chief Technology Officer
+    role: Seasons CTO | Tech Aficionado
```
Comment (Puneeth-n) [inline _data/authors.yml:47]:
```suggestion
    role: Seasoned CTO | Tech Aficionado
```

**Review by Puneeth-n — CHANGES_REQUESTED** [review]

File: _data/authors.yml (hunk 2 above)
Comment (kristianmitk) [reply _data/authors.yml:47]: I was wondering if this was intended 😄 Like switching to ski instructor during winter season ⛷️

Comment (utvara) [reply _data/authors.yml:47]: https://svdictionary.com/words/tech-aficionado

> A person who is insecure about their superficial knowledge in software, hardware and technology in general, but really wants to fit in. It's a description often used in online social blogging bios.

😂 I'm sooo happy I never wrote anything for the blog 😂

Comment (davidsbatista) [reply _data/authors.yml:47]: how can I open a PR for `https://svdictionary.com/words/tech-aficionado` ?

**Pull request merged**